### PR TITLE
Minor Modifications to Developer Guide

### PR DESCRIPTION
**Pull request by iamtr**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -510,11 +508,11 @@ specified otherwise)
     
     Use case resumes at step 1.
 
-**Use case: `US03 - Delete a patient`**
+**Use case: `UC03 - Delete a patient`**
 
 **MSS**
 
-1.  User requests to view patient records(UC01).
+1.  User requests to _view patient records(UC01)_.
```
Comment (ryanlimdx) [inline docs/DeveloperGuide.md]: Should the other lines be formatted the same as well? Cause it only seems to be on one line.

Comment (iamtr) [reply docs/DeveloperGuide.md]: The underline is when referencing other uses cases. The other use cases referenced are being highlighted already.

**Review by ejnan — APPROVED** [review]
LGTM

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -705,28 +702,15 @@ specified otherwise)
 
 #### Non-Functional Requirements
 
-1.  Should work on any _mainstream OS_ as long as it has Java `11` installed.
```
Comment (swtan346) [inline docs/DeveloperGuide.md:708]: I think this is important

**Review by swtan346 — COMMENTED** [review]
LGTM, just add the OS part.

**Pull request merged**